holidayextras/brands
# Globals xl breakpoint

#### What does this PR do? (please provide any background)
1. Adding a larger fixed width container for larger screens.
2. Creating an opportunity for people to use a `make column` mixin for larger screens.
e.g.
```
.product-item {
    .make-sm-column(4, (@grid-gutter-width));
    .make-md-column(3, (@grid-gutter-width));
    .make-lg-column(2.4, (@grid-gutter-width));
    .make-xl-column(2, (@grid-gutter-width));
  }
```
Note : I am not creating a full suite of xl grid classes, although this does create an opportunity for you to make one if you need it.

#### How can this be tested?
`npm link ../brands/brands/globals`

PatternLibrary
Check out my lounges prototype - https://github.com/holidayextras/pattern-library/pull/325 and have a play.

TripApp
Call the mix in in `tripapplite/src/stylesheets/components/structure.less`

#### Any tech debt?
Nope

#### Screenshots / Screencast
<img width="1440" alt="screen shot 2018-02-22 at 16 12 51" src="https://user-images.githubusercontent.com/400087/36549779-82fb2326-17eb-11e8-8445-1a7d0bd04889.png">

#### What gif best describes how you feel about this work?
![](https://media.giphy.com/media/637r7976Qc3HG/giphy.gif)


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.
